src/Opdex.Platform.Application/EntryHandlers/LiquidityPools/GetLiquidityAmountInQuoteQueryHandler.cs

```diff
@@ -22,9 +24,15 @@ public GetLiquidityAmountInQuoteQueryHandler(IMediator mediator)
 
     public async Task<FixedDecimal> Handle(GetLiquidityAmountInQuoteQuery request, CancellationToken cancellationToken)
     {
+        var pool = await _mediator.Send(new RetrieveLiquidityPoolByAddressQuery(request.Pool), cancellationToken);
+        var summary = await _mediator.Send(new RetrieveLiquidityPoolSummaryByLiquidityPoolIdQuery(pool.Id), cancellationToken);
+        if (summary.LockedCrs == UInt256.Zero)
```

Comment:
Can save a database call here by getting the OLPT by liquidity pool address. If OLPT.TotalSupply is 0, there is no liquidity.

```C#
var olpt = await _mediator.Send(new RetrieveTokenByAddressQuery(request.pool), cancellationToken);
if (olpt.TotalSupply == UInt256.Zero) ...
```